packages/card/src/__demo__/index.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright 2017 CA
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+import cardExamples from './examples/cardIndex';
```

Comment:
It is difficult at a glance to identify which examples go with which components.  Using folders might help organization a bit.  I doubt there is a good reason to keep the examples directory flat. 

```
__demo__
    index.js
    examples/
        index.js
        card/
             index.js
            …
        cardAction/
            index.js
            …
        cardBlock/
            index.js
            …
        cardImage/
            index.js
            …
        cardTitle/
            index.js
            …
```